harwest/lib/utils/repository.py

```diff
@@ -26,8 +26,9 @@ def init(self):
         str(config.RESOURCES_DIR.joinpath("readme.template")),
         self.readme_path)
       git.add("README.md")
+      Initial_Commit = config.load_setup_data()['initcommitdate']
       git.commit(message="Initial commit with README.md",
-                 date="Jan/01/2000 00:00", author=self.author)
+                 date=f"{Initial_Commit}", author=self.author)
```

Comment:
How about if we just choose the current date and time for the Initial commit? I think it just might work too without adding this overhead to the user of selecting a date. I believe it'll keep the interface simpler.
Something along the lines of `date=datetime.now().strftime('%b/%d/%Y %H:%M')` Wdyt?

Reply:
right, that would be much better